dev/bots/prepare_package.dart

```diff
@@ -512,21 +517,35 @@ class ArchivePublisher {
   /// This method will throw if the target archive already exists on cloud
   /// storage.
   Future<void> publishArchive([bool forceUpload = false]) async {
-    final String destGsPath = '$gsReleaseFolder/$destinationArchivePath';
+    // Old bucket
+    String destGsPath = '$oldGsReleaseFolder/$destinationArchivePath';
```

Comment:
Could you make this code a loop rather than copied lines, in case we make future changes? Something like:

```
for (final String releaseFolder in <String>[oldGsReleaseFolder, newGsReleaseFolder]) {
```

Reply:
done!